quantum/dynamic_keymap.c

```diff
@@ -35,7 +35,11 @@
 // Explicitly override it if the keyboard uses a microcontroller with
 // more EEPROM *and* it makes sense to increase it.
 #ifndef DYNAMIC_KEYMAP_EEPROM_MAX_ADDR
-#    define DYNAMIC_KEYMAP_EEPROM_MAX_ADDR 1023
+#    ifdef __AVR_AT90USB1286__
```

Comment:
90USB128 actually has 4kB of EEPROM. It may be better to look at `E2END` here ifdef `__AVR__`, for a more general solution.

eg. from iom32u4.h:
![image](https://user-images.githubusercontent.com/4781841/85969612-89f28380-ba0b-11ea-94d8-54201bfd2b32.png)